docs/fdo/fdo-serviceinfo-sys.md

```diff
@@ -0,0 +1,92 @@
+This section defines the “system” module (fdo_sys) specification which provides basic onboarding services
+FDO capable devices. A module is a set of key-value pairs that define the onboarding operations
+that can be performed on given FDO device. Module key-value pairs are exchanged between the device
+and it’s owning Device Management Service. It is up to the owning management service and the
+device to interpret the key-value pairs in accordance to the module specification.
+
+## fdo_sys module definition
+The system module defines basic onboarding operations such as downloading content and executing
+commands. The following table describes key-value pairs for the system module.
+
+| | | |
+|-|-|-|
+| **Key Name** | **Value** | **Meaning** |
+| fdo_sys:active | CBOR Bool | True to activate the fdo_sys module. |
+| fdo_sys:exec | CBOR array of string command arguments. | Performs a system call. |
+| fdo_sys:filedesc | CBOR String | Describes a path to a file that will be used in subsequent file operations. |
+| fdo_sys:write | CBOR BSTR | Appends a block of data to the current file description. |
+
+### fdo_sys:exec message
+
+The fdo_sys:exec commands performs a system call on the device. The value of this command is a
+CBOR Array of string values. The first string in the array is the command to execute and the
+remaining strings are the arguments to the command. It is expected that it would use “exec”
+(system call) or similar API to execute the command on the device.
+
+| | |
+|-|-|
+| JSON (readable description) | ["/bin/sh","startup.sh"] |
+| CBOR | `82` # array(2) <br> `67` # text(7) <br> `2F62696E2F7368` # "/bin/sh" <br> `6A` # text(10) <br> `737461727475702E7368` # "startup.sh"|
+| C example | execvp(“/bin/sh”,(char* []) {“shartup.sh”}) |
+
+### fdo_sys:filedesc message
+
+The fdo_sys:filedesc command describes the path to a file the will be used as a part of the
+on-boarding process. A zero-length file is expected to exist on the local file system after this
+command is received. If the described file already exists it is truncated to zero length, otherwise
+a zero-length file is created. The file needs to be created its permissions are set to read/write
```

Comment:
created -> created and
permissions are set -> permissions set

Reply:
Updated in 06f5e00